.github/workflows/run-integration-tests.yaml

```diff
@@ -0,0 +1,131 @@
+name: Run E2E integration Tests
+
+# This workflow is meant to be called from other repositories' workflows to run integration checks autonomously.
+# While this workflow makes use of secrets, this workflow DOES NOT pull secrets itself and expects upstream callers to
+# provide the secrets via the github secrets functionality.
+#
+# The caller repository must have secrets set up for (Workload Identity Federation is not available for this suite atm)
+#        LYLE_CLIENT_EMAIL
+#        LYLE_PRIVATE_KEY
+#        FIRECLOUD_ACCOUNT_CLIENT_EMAIL
+#        FIRECLOUD_ACCOUNT_PRIVATE_KEY
+# Instruction for getting these secrets securely from vault to github secrets can be found here:
+# https://docs.google.com/document/d/1JbjV4xjAlSOuZY-2bInatl4av3M-y_LmHQkLYyISYns/edit?usp=sharing
```

Comment:
Any reason not to add these secrets to the Terra UI repository? I imagine the eventual goal is to use GitHub Actions to run tests on PRs/commits as well.

Reply:
Ah, I didn't understand how `workflow_call` works (I was thinking of `workflow_dispatch`).

Reply:
👍 yeah they're similarly named. The overall difference is the the CI run in the caller context, so terra-ui wouldn't be seeing all the CI runs of other repos. basically it's like importing a library.